portfolio/src/main/webapp/script.js

```diff
@@ -80,25 +66,64 @@ yogaReveal.onclick = function() {
 }
 
 
-function getName() {
-    console.log('Fetching Name');
+// To parse JSON into a usable object for javascript
+function getComments() {
+
+    // fetch data from server, parse as json file, then reference coms as an object
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const commentList = document.getElementById("comment-container");
+        console.log("fetching comments");
+        comments.forEach((comments) => {
```

Comment:
Nice use of for each.

To make it less ambiguous I would make the argument for forEach singular so:

`comments.forEach((comment) => ...`